### ISSUE TITLE
Please let us enter lyrics manually in the website

### ISSUE THREAD

**Issue by aneesh1122** [tranxuanthang/lrclib#3]
I make my lyrics in my mobile and it's a very lengthy process to upload both the song and the lyrics to my PC and then run lrcget to publish the lyrics.

Just let us enter the lyrics ourselves please
![Screenshot 2024-04-30 004336](https://github.com/tranxuanthang/lrcget/assets/45353488/5a481fc1-b88e-47f8-887a-6a804f13fc2a)

**Comment by adilurrhmn** [comment]
Yes. I want too mobile version LRCGET to upload synchronised lyrics. I installed the .exe file in my pc but but don't understand anything 😭 its very confusing. Can you share the process of uploading syc lyrics.

**Comment by NellowTCS** [comment]
I made this!
https://htmltoolkit.github.io/LRCGetter/
https://github.com/HTMLToolkit/LRCGetter/

I will probably additionally add a CAPTCHA to the Publish page, to be extra safe, but it uses the `/api/publish/` endpoint and a custom CORS proxy to get it all working in the browser.
